pallets/subtensor/src/subnets/weights.rs

```diff
@@ -102,7 +113,7 @@ impl<T: Config> Pallet<T> {
                 .ok_or(Error::<T>::NoWeightsCommitFound)?;
 
             ensure!(
-                Self::is_reveal_block_range(netuid, *commit_block),
+                Self::is_reveal_valid(*commit_block, netuid),
```

Comment:
getting an error like that ("it's a bad time") is the most confusing thing ever. Please return a very detailed error, saying "too early" or "too late" and detailing the exact extrinsic ids and block numbers involved so that people who need to figure out why a validator failed to set weights have something to go on

Reply:
added debug logs so I don't think so we need to add additional logs

Reply:
I don't think the debug logs are being sent to the client which has to debug the error in their code. Can you return a readable error message all the way to the client, please?